.github/workflows/benches.yml

```diff
@@ -67,6 +67,18 @@ jobs:
       - run: echo "$HOME/.cargo/bin" >> $GITHUB_PATH
       - run: bash scripts/environment/prepare.sh
       - run: echo "::add-matcher::.github/matchers/rust.json"
+      # Install and configure sccache, including telling Cargo to use it.
+      - uses: actions-rs/install@v0.1
+        with:
+          crate: sccache
+          version: latest
+      - name: Start sccache
+        env:
+          SCCACHE_REDIS: ${{ secrets.SCCACHE_REDIS }}
+          SCCACHE_IDLE_TIMEOUT: 0
```

Comment:
Just noting for other readers that this disables the timeout.

Reply:
Worth adding as a comment?